egs2/librispeech/asr1/local/path.sh

```diff
@@ -0,0 +1,3 @@
+if [ -d ${MAIN_ROOT}/tools/s3prl ]; then
```

Comment:
Maybe updating `tools/extra_path.sh` is better?

Reply:
Good idea! Will do it.